File: scientific-center/src/main/java/com/scientific/center/magazine/Author.java
package com.scientific.center.magazine;

import java.util.List;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class Author {

	private Long id;
	private String firstName;
	private String lastName;
	private String country;
	private String city;
	private String emailAddress;
	private Boolean mainAuthor;
	private List<AreaOfInterest> areasOfInterest;

}

File: sc-client/src/app/types/author.ts
import {AreasOfInterest} from './areasOfInterest';

export interface Author {
  id: number;
  firstName: string;
  lastName: string;
  country: string;
  city: string;
  emailAddress: string;
  mainAuthor: boolean;
  areasOfInterest: AreasOfInterest[];

}
